test/sales_analyst_test.rb

```diff
@@ -50,4 +50,14 @@ def test_find_average_items_per_merchant_stdev
     assert_equal 1.0, sa.average_items_per_merchant_standard_deviation
   end
 
+  def test_average_item_price_standard_deviation
+    sales_engine = SalesEngine.from_csv({
+      :items     => "./test/fixtures/item_fixture.csv",
+      :merchants => "./test/fixtures/merchant_fixture.csv",
+    })
+    sa = SalesAnalyst.new(sales_engine)
+
+    assert_equal "", sa.average_item_price_standard_deviation
```

Comment:
making comment as reminder to hit this later. no worries for now lets keep on truckin'.